starknet_py/net/account/account_client.py

```diff
@@ -449,3 +465,36 @@ def parse_call(
         calldata.append(data)
 
     return [calldata, entire_calldata]
+
+
+def invoke_function_by_version(
```

Comment:
`make_invoke_tx`?

Reply:
I think it's less descriptive of what this method actually does.

Reply:
I would err on the side of more descriptive one (`make_...`, `get...` etc.). This does not actually invoke anything